docs/EmbeddedAppIOS.md

```diff
@@ -173,4 +173,4 @@ Live reload and all of the debugging tools will work from the simulator (make su
 
 So under the hood, when `RCTRootView` is initialized, it will try to download, parse and run the bundle file from React Native development server. This means all you need to do is to implement your own container view or view controller for the `RCTRootView` – the `RCTRootView` ingests your bundled JS and renders your React components. Bravo!
 
-You can checkout full source code of a sample application [here](https://github.com/tjwudi/EmbededReactNativeExample).
+You can checkout full source code of a sample application [here](https://github.com/hfossli/ReactNativeIntegration).
```

Comment:
Cool! Likes like the original one has a broken link in the readme: https://github.com/tjwudi/EmbededReactNativeExample